File: 백준/Bronze/2566. 최댓값/최댓값.cc
#include <iomanip>
#include <iostream>
#include <sstream>
#include <string>
#include <vector>

using namespace std;

int main() {
  int N = 9, M = 9;

  vector<vector<int>> A(9, vector<int>(9));

  for (int i = 0; i < N; i++) {
    for (int j = 0; j < M; j++) {
      cin >> A[i][j];
    }
  }

  int max = -1;
  int row, col;

  for (int i = 0; i < N; i++) {
    for (int j = 0; j < M; j++) {
      if (max < A[i][j]) {
        max = A[i][j];
        row = i + 1;
        col = j + 1;
      }
    }
  }

  cout << max << endl;
  cout << row << " " << col;

  return 0;
}

File: 백준/Bronze/2566. 최댓값/최댓값.py
import sys

def main() -> None:
    data = sys.stdin.read().strip().split()
    if not data:
        return
    values = list(map(int, data))
    max_value = max(values)
    idx = values.index(max_value)
    row = idx // 9 + 1
    col = idx % 9 + 1
    print(max_value)
    print(row, col)

if __name__ == '__main__':
    main()
